lib/Paneset/Paneset.js

```diff
@@ -56,11 +57,12 @@ class Paneset extends React.Component {
         registerPane: this.registerPane,
         removePane: this.removePane,
         getContainer: this.getContainer,
+        getTopmostContainer: this.getTopmostContainer,
       },
       panes: [],
       layoutCache: props.initialLayouts,
       style: initStyle,
-      changeType: 'init',
+      changeType: 'init', // eslint-disable-line react/no-unused-state
```

Comment:
Could you please clarify whether comment is needed? Fur curiosity I tried without it and seems like it works just fine.
```suggestion
      changeType: 'init',
```